docs/source/streamflow-public-testnet.md

```diff
@@ -0,0 +1,312 @@
+
+```
+18. Get test LPT
+19. Get test ETH
+```
+
+Select the option to get test LPT (note: the option numbering will be slightly different depending on if the wizard is connected to a node running in broadcaster or orchestrators mode). Upon entering the command in the wizard, you should see a transaction submitted by your node. After the transaction confirms, you can see your updated LPT balance by refreshing the wizard:
+
+```
+*-----------------------------*--------------------------------------------*
+|                 ETH Account | 0xeb3F6d3adaA224aB84679b78376F3D96e8bF5781 |
+*-----------------------------*--------------------------------------------*
+|                 LPT Balance |                       10000000000000000000 |
+*-----------------------------*--------------------------------------------*
+|                 ETH Balance |                        9999925448000000000 |
+*-----------------------------*--------------------------------------------*
+```
+
+## Transcoding
+
+### Activation
+
+In order to start transcoding video on the network and earning fees, your orchestrator node must be active.
+
+An orchestrator (identified by its ETH account address) is active in a round if:
+
+- It is registered on-chain. Registration consists of staking any amount of LPT and self-delegating. Anyone can register an orchestrator on-chain
+- It is in the top 100 orchestrators with the most stake 
+
+The active orchestrator set is locked in at the beginning of each round. Any staking activity that occurs during a round will impact membership of the active set in the following round. So, if an orchestrator accumulates stake such that it is in the top 100 orchestrators with the most stake it will join the active set in the next round. If an orchestrator previously was in the top 100 orchestrators with the most stake, but no longer is in the top 100, it will exit the active set in the next round.
+
+You can register an orchestrator and attempt to join the active set in the next round by selecting the following option in `livepeer_cli`:
+
+```
+13. Invoke multi-step "become an orchestrator"
+```
+
+Upon selecting the option, you should be prompted to:
+
+- Set a `rewardCut` which is the percentage of inflationary LPT rewards that you will keep (the rest will be shared with your delegators)
+- Set a `feeShare` which is the percentage of ETH transcoding fees that you will share with your delegators (the rest you will keep)
+- Set a `pricePerPixel` which is the price that you charge (denominated in wei) per pixel of transcoding. See the [configuring payment parameters](#configuring-payment-parameters) for more details on pricing per pixel
+- Set an amount of LPT to stake and self-delegate
+- Set a `serviceURI` which is the public IP and port that broadcasters can send requests and video to be transcoded
```

Comment:
Do you think we should make it clear early in the docs that it's required that you run your orchestrator on a publicly accessible URL? I just worry it will get a little lost in this detial.

Reply:
Added more details about what it means for the `serviceURI` to be publicly accessible in 3fc48ed based on existing docs.